enarx-keep-sgx-shim/.cargo/config

```diff
@@ -3,5 +3,5 @@ target = "x86_64-unknown-linux-musl"
 rustflags = [
     "-C", "relocation-model=pic",
     "-C", "opt-level=3",
-    "-C", "linker=./cc",
+    "-C", "linker=../cc",
 ]
```

Comment:
Just a note that we can probably build the shim on any `libc` since we enforce `-nostdlib`. `debt` `enhancement`

Reply:
Actually, that probably conflicts with moving the shim to `static-pie` and using the musl relocation.